nebu/cli/main.py

```diff
@@ -84,8 +85,19 @@ def set_verbosity(verbose):
     configure_logging(config)
 
 
+def _version_callback(ctx, param, value):
+    if not value or ctx.resilient_parsing:
+        return
+    working_version = __version__
+    click.echo('Nebuchadnezzar version {}'.format(working_version))
+    ctx.exit()
+
+
 @click.group()
 @click.option('-v', '--verbose', is_flag=True, help='enable verbosity')
+@click.option('--version', callback=_version_callback, is_flag=True,
+              expose_value=False, is_eager=True,
+              help='Show the version and exit')
```

Comment:
There is an `@click.version_option(...)` decorator. I've chose not to use thatt because the additional request of #10 would need to do release lookups in this callback structure.